web/src/search_suggestion.ts

```diff
@@ -29,19 +27,10 @@ export type UserPillItem = {
 type TermPattern = Omit<NarrowTerm, "operand"> & Partial<Pick<NarrowTerm, "operand">>;
 
 export type Suggestion = {
-    description_html: string;
+    // description_html is only needed for lines that have a single suggestion
```

Comment:
Is this the best way to explain this to someone not familiar with this file?

I think this needs to provide more context.

Reply:
Good call. Updated to this:

```js
export type Suggestion = {
    // When there's a single pill on a suggestion line, we have space to provide
    // help text (description_html) explaining what the operator does. If a
    // suggestion will be parsed into multiple pills, then we don't include
    // `description_html`.
    description_html: string | undefined;
    search_string: string;
};
```